flutter_src/lib/presentation/navigation/main_navigator.dart

```diff
@@ -35,11 +35,11 @@ class _MainNavigatorState extends State<MainNavigator> {
             tabs: [
               Tab(
                 icon: Icon(Icons.add),
-                text: 'New Macro',
+                text: 'New Chatbots',
```

Comment:
We have to be really careful with what we promise. Using Chatbot as the keyword can give users the impression that we allow them to create full bots which is not the case.

Reply:
True, I changed it back to bots.